FunctionSignatureMap.php

```diff
@@ -304,7 +304,7 @@
 'array_key_first' => ['int|string|null', 'array'=>'array'],
 'array_key_last' => ['int|string|null', 'array'=>'array'],
 'array_keys' => ['array<int,string>|array<int,int>', 'input'=>'array', 'search_value='=>'mixed', 'strict='=>'bool'],
-'array_map' => ['array', 'callback'=>'?callable', 'input1'=>'array', '...args='=>'array'],
+'array_map' => ['array', 'callback'=>'callable', 'input1'=>'array', '...args='=>'array'],
```

Comment:
Phan may deliberately diverge from psalm here, support for null $callback is documented on https://www.php.net/array_map

> An interesting use of this function is to construct an array of arrays, which can be easily performed by using NULL as the name of the callback function

```php
<?php
$a = array(1, 2, 3, 4, 5);
$b = array("one", "two", "three", "four", "five");
$c = array("uno", "dos", "tres", "cuatro", "cinco");

$d = array_map(null, $a, $b, $c);
print_r($d);
?>
```

Reply:
Ah thanks, I'll update